### ISSUE TITLE
list_setups python API call has a default size of 0

### ISSUE THREAD

**Issue by d-behi** [openml/openml-python#475]
#### Description
setups.list_setup has a default size of 0 and unless specified the API call returns an empty dictionary.
This issue is introduced in version 0.7.0 (I tested on 0.6.0 and it worked fine) 
#### Steps/Code to Reproduce

Example:
```python
import openml
openml.setups.list_setups(flow=6969)
# returns {}
openml.setups.list_setups(flow=6969, size = 1)
# returns {2746828: <openml.setups.setup.OpenMLSetup at 0x11b3febd0>}

openml.setups.list_setups(setup=[4151172])
# returns {}
openml.setups.list_setups(setup=[4151172],size = 1)
# returns {4151172: <openml.setups.setup.OpenMLSetup at 0x11a54a950>}
```

#### Expected Results
{2746828: <openml.setups.setup.OpenMLSetup at 0x11b3febd0>}
{4151172: <openml.setups.setup.OpenMLSetup at 0x11a54a950>}

#### Actual Results
{}
{}

#### Versions
```
Darwin-17.5.0-x86_64-i386-64bit
('Python', '2.7.15 (default, May 28 2018, 14:20:45) \n[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.1)]')
('NumPy', '1.13.3')
('SciPy', '1.0.0')
('Scikit-Learn', '0.18.1')
('OpenML', '0.7.0')
```

**Comment by janvanrijn** [comment]
Thanks for reporting this. 

Can you check whether this is still the case in the development branch? 
Recently I fixed a similar issue #463

**Comment by d-behi** [comment]
This is indeed solved in the develop branch. 
I'm closing the issue.
